File: 168_ExcelSheetColumnTitle.py
class Solution(object):
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        strList = []
        while n:
            strList.append(chr((n - 1) % 26 + ord('A')))
            n = (n - 1) / 26
        strList.reverse()
        return "".join(strList)
		

return "" if num == 0 else self.convertToTitle((num - 1) / 26) + chr((num - 1) % 26 + ord('A'))

File: 168_ExcelSheetColumnTitle.cpp
//4.47%
class Solution {
public:
    string convertToTitle(int n) {
		if(!n)
			return "";
		string str("");
		str += (n - 1) % 26 + 'A';
		string strLeft = convertToTitle((n - 1) / 26);
		return strLeft + str;
    }
};

//4.47%
class Solution {
public:
    string convertToTitle(int n) {
		string res("");
		while(n){
			char right = (n - 1) % 26 + 'A';
			res = right + res;
			n = (n - 1) / 26;
		}
		return res;
    }
};

return n == 0 ? "" : convertToTitle((n - 1) / 26) + (char) ((n - 1) % 26 + 'A');